game/hud/src/widgets/Warband/services/session/layout.ts

```diff
@@ -56,35 +61,94 @@ export function savePosition(name: string, pos: Position): LayoutAction {
 
 // async
 
-let defaultWidgets = () => {
+const defaultWidgets = () : any => {
+  // this needs to return positions as % based on 1440x900 (default window size)
+  const w: number = 1440, h: number = 900;
+  alert('defaultWidgets widgets/Warbadn/services/session');
   return {
-    'PlayerHealth': {x:100,y:300,scale:1},
-    'TargetHealth': {x:600,y:300,scale:1},
-    'Party': {x:100,y:10,scale:1},
+    'PlayerHealth': { x: 100/w, y: 300/h, height: 200/h, width: 350/w, scale:1 },
+    'TargetHealth': { x: 600/w, y: 300/h, height: 200/h, width: 350/w,  scale:1 },
+    'Warband': { x: 100/w, y: 10/h , scale:1, height: 200/h, width: 350/w },
+    'Party': { x: 100/w, y: 10/h, scale:1, height: 200/h, width: 350/w },
```

Comment:
Nope.  Ignore this file.  Just a placeholder for me.  @Mehuge did not need to even edit it.